26-register_registration/creator_begins_registering_a_registration.feature

```diff
@@ -66,4 +77,12 @@ Feature: Creator begins registering a Registration
     And they click Expansion panel for Link to resource
     When they enter https://link.springer.com/article/10.1007/s13201-020-01350-9
     And click Search
-    Then they see metadata about the Link in the Expansion panel
\ No newline at end of file
+    Then they see metadata about the Link in the Expansion panel
+
+  @2370
+  Scenario: Creator begins registering with a Link with data from LD+/JSON tag
+    Given Creator begins registering a Registration
+    And they expand the Expansion panel for Link to resource
```

Comment:
ser det står click isteden for expand på de tre featurene over. Kanskje det ikke trengs å endres allikevel? eller evt endre de andre 3 til expand?

Reply:
kan endre de andre også